scripts/e2e_local.sh

```diff
@@ -1,10 +1,14 @@
-#!/bin/sh
+#!/bin/bash
+
+set -eou pipefail
+
 act -j build-push
 
 public_key=$(grep "ATLAS_PUBLIC_KEY" .actrc | cut -d "=" -f 2)
 private_key=$(grep "ATLAS_PRIVATE_KEY" .actrc | cut -d "=" -f 2)
 org_id=$(grep "ATLAS_ORG_ID" .actrc | cut -d "=" -f 2)
-image=$(grep "DOCKER_REPO" .actrc | cut -d "=" -f 2):$(git rev-parse --abbrev-ref HEAD)-$(git rev-parse --short HEAD)
+# this is the format how it's pushed by act -j build-push
+image=$(grep "DOCKER_REGISTRY" .env | cut -d "=" -f 2)/mongodb-atlas-kubernetes-operator:$(git rev-parse --abbrev-ref HEAD)-$(git rev-parse --short HEAD)
```

Comment:
am I missing something? why do we remove DOCKER_REPO if it is still a .env property and it is used in other workflows? for local testing I usually use my own name and for debug workflow it is also useful since I can use my repo name and test full run

Reply:
Discussed and clarified together, tell me Svetlana if you have more questions!